lunes_cms/cms/migrations/0006_convert_group_api_key.py

```diff
@@ -60,7 +60,9 @@ class Migration(migrations.Migration):
             name="token",
             field=models.CharField(
                 default=lunes_cms.cms.models.group_api_key.generate_default_token,
-                help_text="10-50 characters, only digits and upper case letters allowed.",
+                help_text=(
+                    "10-50 characters, only digits and upper case letters allowed."
+                ),
```

Comment:
Normally, migration files will be hashed and compared when deployed. Will these changes introduced by `black` lead to problems during deployment?

Reply:
AFAICT Django does not hash migration files.

Reply:
```
lunes=# SELECT * FROM django_migrations;
 id |          app           |                         name                         |            applied            
----+------------------------+------------------------------------------------------+-------------------------------
  1 | contenttypes           | 0001_initial                                         | 2022-06-07 21:29:43.832816+00
  2 | auth                   | 0001_initial                                         | 2022-06-07 21:29:43.920228+00
```
The database just stores migration IDs.